test/libsolidity/smtCheckerTests/operators/named_arguments_in_any_order.sol

```diff
@@ -0,0 +1,14 @@
+pragma experimental SMTChecker;
+contract C {
+	function f(uint u, string memory s, bool b) internal {}
+
+	function call() public {
+		f({s: "abc", u: 1,     b: true});
+		f({s: "abc", b: true,  u: 1});
+		f({u: 1,     s: "abc", b: true});
+		f({b: true,  s: "abc", u: 1});
+		f({u: 1,     b: true,  s: "abc"});
+		f({b: true,  u: 1,     s: "abc"});
+	}
+}
+// ----
```

Comment:
Why this test doesn't have any assertions? Just for checking that there are no warnings?

Reply:
Yea these were copied from the syntaxTests, used to crash